typings.d.ts

```diff
@@ -0,0 +1,105 @@
+
+/**
+ * The plugin options
+ */
+interface HtmlWebpackPluginOptions {
```

Comment:
There is already a handwritten interface that might be more accurate:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4393935ba72a19e07211b983897d6a8dd2254a99/types/html-webpack-plugin/index.d.ts#L36

In the long run it would be amazing if we can generate this interface from code jsdoc itself.

Reply:
Is it possible to export a jsdoc typedef?